x-pack/plugins/lens/public/xy_visualization/reference_line_helpers.tsx

```diff
@@ -109,7 +109,8 @@ export function getStaticValue(
       filteredLayers,
       accessors,
       activeData,
-      groupId !== 'x' // histogram axis should compute the min based on the current data
+      groupId !== 'x', // histogram axis should compute the min based on the current data
+      groupId
```

Comment:
you pass the groupId here and then pass it down from `computeStaticValueForGroup` to `computeOverallDataDomain` and then use it as `groupId !== 'x'` - like a previous parameter in this function. Can't you just take it's value and pass down? I mean using `minZeroOrNegativeBase`.

Reply:
I thought about that, but I think these two should really be two separate configuration things, and I'm not sure I would like to encapsulate this type of information in the function at lower level.
As alternative fix I thought also to pass another boolean flag `avoidStacking`, but then I came back to this raw solution. Maybe the boolean flag is best also in this case?

Reply:
Ok, I get it, that's more descriptive this way. I am leaning towards adding another boolean, but in the end it's not a big deal so I am leaving the decision to you :)